test/GroupByTest.cpp

```diff
@@ -788,6 +795,8 @@ TEST(GroupBy, AliasResultReused) {
                   {Variable{"?a"}},
                   {std::move(alias1), std::move(alias2)},
                   std::move(values)};
+  groupBy.createRuntimeInfoFromEstimates(
+      std::make_shared<RuntimeInformation>());
```

Comment:
I think these all could be made unnecessary.

Reply:
(but maybe we have to set `isRoot` automatically, or maybe we find a non-intrusive way to fix this automagically.